packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -68,6 +73,28 @@ export interface IDocumentSchema {
 	refSeq: number;
 
 	runtime: Record<string, DocumentSchemaValueType>;
+
+	// Other document related properties that are not related to the schema.
```

Comment:
I'd say it somethinkg like this.  It feels weird to say "not related to the schema", based on the naming of types, although maybe that's not untrue

```suggestion
	// Info about this document that can be updated via Document Schema op, but isn't required to be understood by all clients (unlike the rest of IDocumentSchema)
```